src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3Connection.cs

```diff
@@ -145,7 +145,7 @@ private void CheckForShutdown()
 
                 _ = _connectionClosedTask.ContinueWith(closeTask =>
                 {
-                    if (closeTask.IsFaulted && NetEventSource.IsEnabled)
+                    if (closeTask.IsFaulted && NetEventSource.Log.IsEnabled())
```

Comment:
If the clauses here aren't swapped, will the linker end up making this `if (closeTask.IsFaulted) { }` or will it be able to eliminate the whole if block, regardless?

Reply:
The linker will eliminate the conditionally guarded block, any conditions guarding it will be untouched.